src/cpp/src/text2image/stable_diffusion_xl_pipeline.hpp

```diff
@@ -67,32 +67,22 @@ class Text2ImagePipeline::StableDiffusionXLPipeline : public Text2ImagePipeline:
 
         const std::string text_encoder = data["text_encoder"][1].get<std::string>();
         if (text_encoder == "CLIPTextModel") {
-            AdapterConfig adapters;
-            std::filesystem::path path = root_dir / "text_encoder";
-            if(update_adapters_from_properties(properties, adapters) && !adapters.get_tensor_name_prefix()) {
-                auto clip_properties = properties;
-                adapters.set_tensor_name_prefix("lora_te1");
-                clip_properties[ov::genai::adapters.name()] = adapters;
-                m_clip_text_encoder = std::make_shared<CLIPTextModel>(path, device, clip_properties);
-            } else {
-                m_clip_text_encoder = std::make_shared<CLIPTextModel>(path, device, properties);
-            }
+            m_clip_text_encoder = std::make_shared<CLIPTextModel>(
+                root_dir / "text_encoder",
+                device,
+                properties_for_text_encoder(properties, "lora_te1")
```

Comment:
why the same is not required for SD pipeline ?

Reply:
Because CLIPTextModel has default prefix "lora_te" that  matches with what we have in lora adapter for SD.